charts/harmony-chart/templates/NOTES.txt

```diff
@@ -26,7 +26,7 @@ To connect to the dashboard, start port-forwarding with the following command:
 Now you can connect to https://localhost:8443. The certificate is self-signed by
 the cluster.
 {{- end }}
-
+{{ end }}
```

Comment:
This closes over two `if` statements:
```
{{ if and .Values.prometheusstack.grafana.enabled .Values.prometheusstack.grafana.ingress.enabled }}
 ...
  {{- if .Values.k8sdashboard.enabled }}
    ...
  {{- end }}
{{ end }}
```

I'm guessing k8s dashboard is not dependent on prometheus stack, so I think this `{{ end }}` should be moved up to make to independent `if`s:
```
{{ if and .Values.prometheusstack.grafana.enabled .Values.prometheusstack.grafana.ingress.enabled }}
  ...
{{ end }}
{{- if .Values.k8sdashboard.enabled }}
  ...
{{- end }}
```

Reply:
Oop, good catch.

Reply:
Oh. Nice! Puched a fix for that.

Reply:
@mtyaka any objections to merge it as @bradenmacdonald already approved it?